Electron-Basic-Video-Chat/README.md

```diff
@@ -0,0 +1,160 @@
+
+session.on({
+  streamCreated: (event) => {
+    session.subscribe(event.stream, 'subscriber', (error) => {
+      if (error) console.log(`There was an issue subscribing to the stream ${event}`);
+    });
+  }
+});
+
+## Publishing an audio video stream to the session
+
+Upon successfully connecting to the OpenTok session (see the previous section), the application
+initializes an OpenTok Publisher object and publishes an audio-video stream to the session. This is
+done inside the completion handler for the connect() method, since you should only publish to the
+session once you are connected to it.
+
+The Publisher object is initialized as shown below. The `OT.initPublisher()` method takes three
+optional parameters:
+
+* The target DOM element or DOM element ID for placement of the publisher video
+* The properties of the publisher
+* The completion handler
+
+```
+const publisher = OT.initPublisher('publisher', (error) => {
+  if (error) {
+    return console.log('There was an error initializing the publisher', error);
+  }
+});
+
+```
+
+Once the Publisher object is initialized, we publish to the session using the `publish()`
+method of the Session object:
+
+session.publish(publisher, (error) => {
+  if (error) {
+    return console.log('There was an error publishing', error);
+  }
+});
+## Subscribing to another client's audio-video stream
+
+The Session object dispatches a `streamCreated` event when a new stream (other than your own) is
+created in a session. A stream is created when a client publishes to the session. The
+`streamCreated` event is also dispatched for each existing stream in the session when you first
+connect. This event is defined by the StreamEvent object, which has a `stream` property,
+representing stream that was created. The application adds an event listener for the
+`streamCreated` event and subscribes to all streams created in the session using the
+`Session.subscribe()` method:
+
+    // Subscribe to a newly created stream
+    session.on({
+      streamCreated: (event) => {
+        session.subscribe(event.stream, 'subscriber', (error) => {
+          if (error) console.log(`There was an issue subscribing to the stream ${event}`);
+        });
+      }
+    });
+
+The `Session.subscribe()` method takes four parameters:
+
+* The Stream object to which we are subscribing to
+* The target DOM element or DOM element ID (optional) for placement of the subscriber video
+* A set of properties (optional) that customize the appearance of the subscriber view
+* The completion handler function (optional) that is called when the method completes
+  successfully or fails
+
+## Running the application
+
+`$ npm start`
+
+## Known Issues
+
+* When starting the application, you'll notice a silent error: 
+`ERROR:audio_send_stream.cc(93)] Failed to set up send codec state.`
+This is a known [Electron issue](https://github.com/electron/electron/issues/8991).
```

Comment:
Can you add next steps for the reader? If it's a known issue, what do they do next? e.g. wait for electron team to resolve, a workaround, restart?

Reply:
fixed in the latest commit